deploy.sh

```diff
@@ -18,11 +18,15 @@ echo "Fetching data…"
 git clone --depth 1 https://github.com/datenanfragen/data data_tmp
 
 echo "Creating directories…"
-for lang in ${languages[@]}
-do
+
+create_directory(){
+    lang=$1
     mkdir -p "content/$lang/company"
     mkdir -p "content/$lang/supervisory-authority"
-done
+}
+
+export -f create_directory
+parallel create_directory ::: ${languages[@]}
```

Comment:
You don't need to do that for this. Seems a little overboard for creating a couple of directories. :D